llpc/context/llpcCompiler.cpp

```diff
@@ -2330,116 +2334,60 @@ Result Compiler::buildRayTracingPipelineInternal(RayTracingContext &rtContext,
   // Record which module calls TraceRay(), except the first one (For indirect mode, it is the entry function which will
   // never call TraceRay(). For inlined mode, we don't need to care).
   std::vector<bool> moduleCallsTraceRay;
+  std::unique_ptr<Module> gpurtShaderLibrary;
+  if ((shaderInfo.size() - pipelineInfo->shaderCount) > 1) {
+    gpurtShaderLibrary = std::move(modules.back());
+    modules.pop_back();
+    shaderInfo = shaderInfo.drop_back();
+  }
```

Comment:
Could you please add a comment on why the if condition detects GPURT? It being last seems to be an internal convention?

Maybe it would be nicer to have an `IsGpurt(const Module&)` helper, but it's clear how to do that in a reliable way (i.e., without relying on particular functions being present and not being renamed).